src/components/test/common-tests.js

```diff
@@ -151,34 +151,52 @@ export function testPresentationalComponent(
 /**
  * Set of tests common to all base components
  *
- * @param {FunctionComponent} Component
+ * @param {import('preact').FunctionComponent} Component
```

Comment:
Most of the changes in this module are me finally getting around to extending the "base" common tests to take and use the options object that the other common test suites already use. I'll call out where there's actually a change.